.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,22 @@
+## For issue
+
 ## Please verify the following:
 
 - [ ] `yarn test` **jest** tests pass with new tests, if relevant
 - [ ] `yarn lint` **eslint** checks pass with new code, if relevant
 - [ ] `yarn format:check` **prettier** checks pass with new code, if relevant
 - [ ] `README.md` (or relevant documentation) has been updated with your changes
+- [ ] Include issue number in title and description
```

Comment:
For 2, when I say description, I meant the first part of the template. I'm fine with the change you suggested above.

Reply:
Also, I really like the issue number in the title. It makes it easier to search from the PR list, and it forces the creator to match with an issue.

Reply:
Agreed, I haven't noticed that field. 

Ok so then, let's rely on that plus the comment added in this PR. 
Should we update the comment from:

`<!-- If this PR addresses an issue, link to it in description: "Addresses #333" -->`

to 

`<!-- If this PR addresses an issue, link to it in description: "Closes #333" -->`

Reply:
Using the `Closes #xyz` will label it under the "Development" field for you, I use this quite a bit. Sometimes the Development field makes you choose the repo (unsure why, as you're already there) and then find the issue name. Either work, but I know that syntax works - unsure about Addresses

As far as an issue for each PR, we don't always have them. Sometimes an idea pops in your head and you just go for it (or have discussed it with the team if it needs discussing). So for IR folks, I don't think it is as much of a requirement.

For outside contributions though, it would be nice to understand what the issue is that they are trying to solve, but that of course could be done with a good PR write up as well, not really a requirement for me although low gafo

Reply:
I agree! We leave it as a suggestion if the issue exists. Changed the working from "Addresses" to "Closes".